products/proxmox-ve.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Proxmox VE
+category: os
+iconSlug: proxmox
+permalink: /proxmox-ve
+alternate_urls:
+-   /pve
+-   /proxmoxve
+activeSupportColumn: Supported
+releasePolicyLink: https://pve.proxmox.com/wiki/Roadmap
+releaseDateColumn: true
+eolColumn: true
+discontinuedColumn: false
+versionCommand: pveversion –v
+
+releases:
+-   releaseCycle: "7.x"
+    support: true
+    eol: false
+    latest: "7.3"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_7.3
+    releaseDate: 2021-07-06
+-   releaseCycle: "6.x"
+    support: false
+    eol: 2022-07-01
+    latest: "6.4"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_6.4
+    releaseDate: 2019-07-16
+-   releaseCycle: "6.x"
+    support: false
+    eol: 2020-07-01
+    latest: "5.4"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_5.4
+    releaseDate: 2017-04-07
+-   releaseCycle: "4.x"
+    support: false
+    eol: 2018-06-01
+    latest: "4.4"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_4.4
+    releaseDate: 2015-05-10
+-   releaseCycle: "3.x"
+    support: false
+    eol: 2017-02-01
+    latest: "3.4"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_3.4
+    releaseDate: 2013-05-24
+-   releaseCycle: "2.x"
+    support: false
+    eol: 2014-02-01
+    latest: "2.3"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_2.3
+    releaseDate: 2012-05-04
+-   releaseCycle: "1.x"
+    support: false
+    eol: 2013-01-01
+    latest: "1.9"
+    link: https://pve.proxmox.com/wiki/Roadmap#Proxmox_VE_1.9
+    releaseDate: 2008-10-29
+
+---
+
+> [Proxmox Virtual Environment (PVE)](https://www.proxmox.com/en/proxmox-ve) Proxmox VE is a complete, open-source server management platform for enterprise virtualization. It tightly integrates the KVM hypervisor and Linux Containers (LXC), software-defined storage and networking functionality, on a single platform. With the integrated web-based user interface you can manage VMs and containers, high availability for clusters, or the integrated disaster recovery tools with ease.
```

Comment:
A shorter and more neutral definition would be best, such as the one from Wikipedia :

> Proxmox Virtual Environment is an open-source software server for virtualization management. It is a hosted hypervisor that can run operating systems including Linux and Windows on x64 hardware.